lms/templates/ccx/grading_policy.html

```diff
@@ -3,8 +3,8 @@
 <div id="warn-coach" class="wrapper-msg urgency-high warning">
   <div class="msg">
     <i class="msg-icon icon-warning-sign"></i>
-    <div class="msg-content">
-      <h3 class="title">${_("WARNING")}</h3>
+    <div class="msg-content" tabindex="0">
+      <h3 class="title" role="heading">${_("WARNING")}</h3>
```

Comment:
@amir-qayyum-arbisoft, this is unnecessary.  An ```<h3>``` is a heading, so it has an implied role of heading.  Explicitly defining that is frowned upon:
https://w3c.github.io/aria-in-html/#aria-adds-nothing-to-default-semantics-of-most-html-elements